guillotina/_settings.py

```diff
@@ -91,5 +91,7 @@
     "search_parser": "default",
     "object_reader": "guillotina.db.reader.reader",
     "thread_pool_workers": 32,
+    "keep_alive": 5,
```

Comment:
5 is very low and I'm not sure it'll actually help the situation if this is the same problem that I've seen.

fwiw, nginx defaults to 75sec keep alive I believe.

Reply:
default keep alive timeout at uvicorn is 5 (https://github.com/encode/uvicorn/blob/master/uvicorn/config.py#L134)